src/main/java/org/folio/circulation/item/service/CirculationItemService.java

```diff
@@ -1,11 +1,15 @@
 package org.folio.circulation.item.service;
 
-import org.folio.rs.domain.dto.CirculationItem;
+import org.folio.circulation.item.domain.dto.CirculationItem;
 
 public interface CirculationItemService {
     CirculationItem getCirculationItemById(String id);
 
-    CirculationItem createCirculationItem(String circulationItemId, org.folio.rs.domain.dto.CirculationItem circulationItem);
+    CirculationItem getCirculationItemByBarcode(String barcode);
+
+    CirculationItem createCirculationItem(String circulationItemId, org.folio.circulation.item.domain.dto.CirculationItem circulationItem);
```

Comment:
Since you imported org.folio.circulation.item.domain.dto.CirculationItem, so I believe, it would be better to remove the package reference and use just the CirculationItem class name as an input param type.

Reply:
Absolutely, fixed now